src/Components/Onboarding/Steps/Artists/PopularArtists.tsx

```diff
@@ -24,16 +24,18 @@ class PopularArtistsContent extends React.Component<Props, any> {
   }
 
   onArtistFollowed(artistId: string, store: RecordSourceSelectorProxy, data: SelectorData): void {
-    const suggestedArtist = store.get(data.followArtist.artist.related.suggested.edges[0].node.__id)
-    this.excludedArtistIds.add(suggestedArtist.getValue("_id"))
+    const suggestedArtistEdge = data.followArtist.artist.related.suggested.edges[0]
+    const popularArtist = data.followArtist.popular_artists.artists[0]
+    const artistToSuggest = store.get(((suggestedArtistEdge && suggestedArtistEdge.node) || popularArtist).__id)
+    this.excludedArtistIds.add(artistToSuggest.getValue("_id"))
```

Comment:
Not sure if you saw this, so pasting here again https://github.com/artsy/reaction/pull/466#discussion_r159528295